static/js/click_handlers.js

```diff
@@ -625,7 +633,7 @@ exports.initialize = function () {
             e.stopPropagation();
             // Cancel editing description if Escape key is pressed.
             if (e.which === 27) {
-                $("[data-finish-editing='.stream-description-editable']").hide();
+                $("[data-finish-editing='.description-editable']").hide();
```

Comment:
Is changing this to `.description-editable` still works? There are no changes in templates, then shouldn't this break when we find with `[data-finish-editing='.stream-description-editable']`? 
I was expecting this to `.stream-description-editable` in case of stream and `.group-description-editable` in case of user groups, as per template.

Reply:
yeah earlier I was thinking to change it to change it to `description-editable` to make it generic but left it in midway and forgot it, also I was not aware of the bug it is causing ( when we click edit-description and then esc to cancel it the checkmark remains visible), used an if statement to seprate it.